lib/adapters/class-adapter.php

```diff
@@ -130,6 +130,15 @@ private function add_aggregation( Aggregation $aggregation ): void {
 		$this->aggregations[ $aggregation->get_query_var() ] = $aggregation;
 	}
 
+	/**
+	 * Adds a custom Aggregation to the list of active aggregations.
+	 *
+	 * @param Aggregation $aggregation The aggregation to add.
+	 */
+	public function add_custom_aggregation( Aggregation $aggregation ): void {
```

Comment:
It would be simpler to make the existing `add_aggregation` method public, so let's do that instead

Reply:
This still needs to be done.

Reply:
Sorry forgot about committing changes, done now